src/pg_to_evalscript/javascript_processes/last.js

```diff
@@ -0,0 +1,17 @@
+function last(arguments) {
+  const { data, ignore_nodata = true } = arguments;
+
+  if (data === null || data === undefined) {
+    throw new Error("Mandatory argument `data` is either null or not defined.");
+  }
+
+  if (data.length === 0) {
+    return null;
+  }
+
+  if (ignore_nodata) {
+    return [...data].reverse().find((x) => x !== null);
+  }
+
+  return [...data].reverse()[0];
```

Comment:
I think there is more efficient way to get last element of an array.

Reply:
Should be fixed

Reply:
Sorry, I should have been more clear about it. What I had in mind is good old 
```
data[data.length-1] 
```
fftr

Reply:
According to google, `pop()` is supposed to be the fastest/most efficient way to get the last element of the array, so I went with this.